### ISSUE TITLE
SQL Tools extension won't load

### ISSUE THREAD

**Issue by mattmcnabb** [microsoft/sqltoolsservice#2001]
Here's an error I recently started getting when attempting to connect:

![image](https://user-images.githubusercontent.com/6998507/231800137-8dceff52-d780-485e-828d-1dd44849a435.png)

OS Name:                   Microsoft Windows Server 2016 Standard
OS Version:                10.0.14393 N/A Build 14393
VS Code Version: 1.77.3
MSSQL Extension Version: 1.18.0

I see this in the output pane during initialization:

```Initializing SQL tools service for the mssql extension.
Note: mssql commands will be available after installing the service.

Platform: win32, x86_64 (Windows)

Installing SQL tools service to c:\Users\mmcnabb\.vscode\extensions\ms-mssql.mssql-1.18.0\sqltoolsservice\4.5.1.6\Windows.
Downloading https://github.com/Microsoft/sqltoolsservice/releases/download/4.5.1.6/microsoft.sqltools.servicelayer-win-x64-net7.0.zip
[ERROR] Error: Request error: ECONNREFUSED
```

**Comment by mattmcnabb** [comment]
I should mention, I have no trouble downloading this ZIP file on the same computer in a browser. Is there possibly a VS Code setting at play?

**Comment by mattmcnabb** [comment]
@Charles-Gagnon yes that's likely, but navigating directly to the URL in that error worked fine, on the same computer.

**Comment by Charles-Gagnon** [comment]
You probably have your browser set up with the correct proxy settings, in VS Code you'll need to do that manually.

There's a setting `http.proxy` that you can set up with your proxy information, after which the download should work.

e.g. `"http.proxy": "https://userName@fqdn:yourPassword@yourProxyURL.com:8080"`

https://code.visualstudio.com/docs/setup/network also details some other ways of handling proxy support.
